ctapipe_io_lst/__init__.py

```diff
@@ -40,6 +38,60 @@
     num_mirror_tiles=198,
 )
 
+DRAGON_COUNTERS_DTYPE = np.dtype([
+    ('pps_counter', np.uint16),
+    ('tenMHz_counter', np.uint32),
+    ('event_counter', np.uint32),
+    ('trigger_counter', np.uint32),
+    ('local_clock_counter', np.uint64),
+]).newbyteorder('<')
+
+
+TIB_DTYPE = np.dtype([
+    ('event_counter', np.uint32),
+    ('pps_counter', np.uint16),
+    ('tenMHz_counter', np.uint32),
+    ('stereo_pattern', np.uint8),
+    ('masked_trigger', np.uint8),
+]).newbyteorder('<')
+
+CDTS_AFTER_37201_DTYPE = np.dtype([
+    ('timestamp', np.uint64),
+    ('address', np.float32),
+    ('event_counter', np.float32),
+    ('busy_counter', np.float32),
+    ('pps_counter', np.float32),
+    ('clock_counter', np.float32),
+    ('trigger_type', np.uint8),
+    ('white_rabbit_status', np.uint8),
+    ('stereo_pattern', np.uint8),
+    ('num_in_bunch', np.uint8),
+    ('cdts_version', np.uint32),
+]).newbyteorder('<')
+
+CDTS_BEFORE_37201_DTYPE = np.dtype([
+    ('event_counter', np.float32),
+    ('pps_counter', np.float32),
+    ('clock_counter', np.float32),
+    ('timestamp', np.float64),
+    ('camera_timestamp', np.float64),
+    ('trigger_type', np.uint8),
+    ('white_rabbit_status', np.uint8),
+    ('unknown', np.uint8),
```

Comment:
@FrancaCassol do you know what this 8th value here is? It was not used before

Reply:
Hi @MaxNoe, looking back into the documentation, the name of this word was "arbitraryInformation".
In any case, it was never used.